### PR TITLE
[16988] Home bb application

### PR DESCRIPTION
**Pull request by peschina**
[comment]: <> (This file has been added on OGCIO fork)
### Ticket:

- [16988](https://dev.azure.com/OGCIO-Digital-Services/Digital%20Services%20Programme/_workitems/edit/16988)

### Description

Seed HOME building blocks application

## Type

- [ ] **Dependency upgrade**
- [ ] **Bug fix**
- [ ] **New feature**
- [x] **Dev change**

### Checklist:

- [ ] I have added tests that prove my fix is effective or that my feature works

### Screenshots:

N/A

### CODE REVIEW

File: packages/cli/src/commands/database/ogcio/ogcio-seeder-local.json [hunk 1]
```diff
@@ -129,6 +129,16 @@
         "secret": "analytics_app_local_secret",
         "id": "kgz1zomlw27cxx4pxh5gi",
         "is_third_party": false
+      },
+      {
+        "name": "Home Building Block",
+        "description": "Building Blocks home application",
+        "type": "Traditional",
+        "redirect_uri": "http://localhost:3004/callback",
+        "logout_redirect_uri": "http://localhost:3004/dashboard",
```
Comment (nnorbert) [inline packages/cli/src/commands/database/ogcio/ogcio-seeder-local.json:138]: This has to be just the domain without any path. The entry page will decide where to redirect the user. At least this is how the other apps work.

Comment (peschina) [reply packages/cli/src/commands/database/ogcio/ogcio-seeder-local.json:138]: I'll change it

Comment (nnorbert) [reply packages/cli/src/commands/database/ogcio/ogcio-seeder-local.json]: Oh, I see some apps are using the /admin path. Okay, maybe it is good like this then.

**Review by nnorbert — APPROVED** [review]
LGTM!

**Review by alfonsograziano — APPROVED** [review]

**Pull request merged**